packages/components/src/DateTimePickers/DateTime/Picker/Picker.component.test.js

```diff
@@ -46,6 +46,7 @@ describe('DateTime.Picker', () => {
 			useSeconds: true,
 			useTime: true,
 			useUTC: false,
+			t: expect.any(Function),
```

Comment:
You forgot to remove that

Reply:
this is expected to be here, since there is function `t` in the DateTimePicker